src/System.Data.SqlClient/tests/ManualTests/SQL/SqlBulkCopyTest/CopyWidenNullInexactNumerics.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections.Generic;
```

Comment:
Please add License header to the new class.

Reply:
Done